application/models/Question.php

```diff
@@ -774,15 +781,17 @@ private function sortAnswerOptions($answerOptions)
 
         // Sort alphabetically if applicable
         if ($this->shouldOrderAnswersAlphabetically()) {
+            if (is_null($language)) {
+                $language = $this->survey->language;
+            }
             foreach ($answerOptions as $scaleId => $scaleArray) {
                 $sorted = array();
 
-                // We create an aray sorted that will use the answer in the current language as key, and that will store its old index as value
+                // We create an array sorted that will use the answer in the current language as key, and that will store its old index as value
                 foreach ($scaleArray as $key => $answer) {
-                    $sorted[$answer->answerl10ns[$this->survey->language]->answer] = $key;
+                    $sorted[$answer->answerl10ns[$language]->answer] = $key;
```

Comment:
Check if isset ?

I can happen after import TSV or have specific issue …

Reply:
I think this s covered on the comments above.

Reply:
> I think this s covered on the comments above.

Not : _I can happen after import TSV or have specific issue …_ 

If you can not do the database repair : it happen a lot … but no issue with debug = 0

Reply:
Sure. Let's check also then. It doesn't hurt.

Reply:
I check : seems OK with broken language to

Reply:
It break before